openslides/motions/static/js/motions/site.js

```diff
@@ -1469,6 +1469,22 @@ angular.module('OpenSlidesApp.motions.site', [
             }
             PersonalNoteManager.saveNote($scope.motion, $scope.motion.personalNote);
         };
+        $scope.personalNotePinned = false;
+        $scope.pinPersonalNote = function () {
+            $scope.personalNotePinned = !$scope.personalNotePinned;
+            if ($scope.personalNotePinned) {
+                resizePersonalNoteContainer();
+            } else {
+                $('#personalNote').css('width', '');
```

Comment:
Please use AngularJS like DOM manipulation methods and not jQuery.

Reply:
Using ng-style here is not very good: on every resize an angular digest cycle has to be started, so angular can update the directive. This causes many laggs.

Reply:
Hm. I have to check this ...

Reply:
One spontaneous idea:
<tag style="width: {{ width }}px;"></tag>
I'll check if this works better..

Reply:
Ok, this is also angular related. If this attribute should change, a digest cycle is necessary.

I would prefer using just jquery, because always starting a new digest cycle slows down the performance a lot in fact that angular reloads everything in the background. JQuery is grea for just DOM manipulation that should happend fast and responsive and angular should not bother with the width of a fixed tile.

Reply:
Ok. I still don't like it but we can do it that way. Please add a short comment at the top of the code block for this feature (before line 1487 with extra newline) and explain in three words that we use a JQLite instead of AngularJS stuff for simplification at this place. So we later can remember the reason for it.

Reply:
Done.